aip/cloud/25146.md

```diff
@@ -0,0 +1,28 @@
+---
+id: 25146
+state: draft
+created: TBD
+placement:
+  category: gcp
+---
+
+# Generic Fields
+
+Guidelines related to generic fields in [AIP-146](../0146.md) **should** be applied
+with the following exceptions.
+
+## Maps
+
+Maps with nested object types **must** not exist.
```

Comment:
I think the it's supposed to be `**must not**`, so the "not" ends up in the `<strong></strong>`.